findContact/FindContactModal.js

```diff
@@ -180,24 +260,25 @@ class FindContactModal extends React.Component {
                 {renderNewContactBtn()}
               </div>
               <SearchAndSort
-                packageInfo={this.props.packageInfo || packageInfo}
```

Comment:
it's hard to see the changes. Could you please highlight them somehow?

Reply:
Just added one prop ‘columnWidths’ and sorted in alphabetical order.

Reply:
thanks, but I hope you understand that the next time it's better to not sort props in such cases